.gitignore

```diff
@@ -36,4 +36,5 @@ build/*
 .vscode/*
 .cache/*
 
-CMakeCache.txt
+**/config.toml
+/main.sqlite3
```

Comment:
`main.sqlite3` should be `main.db`

Reply:
I changed it to `main.sqlite3`.

Reply:
Please change it back to `main.db` - it's simpler and shorter

Reply:
Isn't `.sqlite3` a bit more descriptive?

Reply:
imo too long

Reply:
if you want to compromise - some ppl use `.db3` but I don't think that's a good idea

Reply:
Okay, I will change it to `.db` then.